orchagent/flexcounterorch.cpp

```diff
@@ -69,12 +72,22 @@ unordered_map<string, string> flexCounterGroupMap =
 
 FlexCounterOrch::FlexCounterOrch(DBConnector *db, vector<string> &tableNames):
     Orch(db, tableNames),
-    m_flexCounterConfigTable(db, CFG_FLEX_COUNTER_TABLE_NAME),
     m_bufferQueueConfigTable(db, CFG_BUFFER_QUEUE_TABLE_NAME),
     m_bufferPgConfigTable(db, CFG_BUFFER_PG_TABLE_NAME),
     m_deviceMetadataConfigTable(db, CFG_DEVICE_METADATA_TABLE_NAME)
 {
     SWSS_LOG_ENTER();
+    m_delayTimer = new SelectableTimer(timespec{.tv_sec = FLEX_COUNTER_DELAY_SEC, .tv_nsec = 0});
+    if (WarmStart::isWarmStart())
+    {
+        auto executor = new ExecutableTimer(m_delayTimer, this, "FLEX_COUNTER_DELAY");
```

Comment:
I am not convinced with this approach. I can use some to understand the approach.

Previously we had hardcoded delay in FC.
The hardcoded delay became a problem and we needed more deterministic delayed approach. So we added fast-reboot new infra which used warm-reboot kind of reconciliation and used fast-reboot done status to enable FC.

But now we again go back to hardcoded delay. I am afraid that to solve the new problem we (in a way) drift towards the day1 problem. Some scenarios that I can image that will now become problematic can be when 60s not being enough and overlaps w/ apply_view. Or similar race conditions that led to more deterministic delay.

Also, I think the concerns that Jingwen raised about new entries showing up in the table will still be present, right?
The config will still look different than what init config would want.

@wen587 , what do you think about this? Does this change really solve the concern that you have raised?